doc/bootstrap.rst

```diff
@@ -51,28 +64,29 @@ This command does a few things:
 Interacting with the cluster
 ============================
 
-You can easily start up a container that has all of the Ceph packages
-installed to interact with your cluster::
+To interact with your cluster, start up a container that has all of 
+the Ceph packages installed::
 
-  sudo ceph-daemon shell --config ceph.conf --keyring ceph.keyring
+  [any monitor] $ sudo ceph-daemon shell --config ceph.conf --keyring ceph.keyring
 
 The ``--config`` and ``--keyring`` arguments will bind those local
 files to the default locations in ``/etc/ceph`` inside the container
 to allow the ``ceph`` CLI utility to work without additional
 arguments.  Inside the container, you can check the cluster status with::
 
-  ceph status
+  [ceph: root@monitor_1_hostname /]# ceph status
 
-In order to interact with the Ceph cluster outside of a container, you
-need to install the Ceph client packages and install the configuration
-and privileged administrator key in a global location::
+In order to interact with the Ceph cluster outside of a container
+(that is, from the command line), you need to install the Ceph 
+client packages and install the configuration and privileged 
```

Comment:
Do you mean to install ceph client packages, the configuration, and privilged administration key in a global location? Or do you mean to say "install the ceph client packages, and then install the configuration and privileged administrator key in a global location"?

Reply:
1. Hi Alex.
2. This sentence, which I inherited and did not generate, intends to say that two conditions must obtain in order for the user to interact with the Ceph cluster from outside a Ceph shell: (1) the ceph.conf and ceph.keyring files must be installed in /etc/ceph/ and (2) the Ceph client packages must be installed on the system. The ambiguity of lists in analytic languages strikes again.
3. Since I've decided that I'll keep this pull request open until all of the raised issues are addressed or the limits of human patience have been reached, I'll rework this sentence.
4. You're not wrong about all of the other places in the text that use second-person or unnecessary future tense, and your suggestions are right. We'll extirpate all of the places in the text where this kind of thing appears, but it might take more than just this one pull request.